examples/vello_editor/src/main.rs

```diff
@@ -11,18 +12,55 @@ use vello::{AaConfig, Renderer, RendererOptions, Scene};
 use winit::application::ApplicationHandler;
 use winit::dpi::LogicalSize;
 use winit::event::*;
-use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop, EventLoopProxy};
 use winit::window::Window;
 
+mod access_ids;
+use access_ids::{TEXT_INPUT_ID, WINDOW_ID};
+
 // #[path = "text2.rs"]
 mod text;
 use parley::{GenericFamily, StyleProperty};
 
+const WINDOW_TITLE: &str = "Vello Text Editor";
+
 // Simple struct to hold the state of the renderer
 pub struct ActiveRenderState<'s> {
-    // The fields MUST be in this order, so that the surface is dropped before the window
+    // The fields MUST be in this order, so that the surface and AccessKit adapter are dropped before the window
```

Comment:
I don't think this is true for wgpu any more, since the `Arc<Window>` is also owned by the `wgpu` Surface.

This might still be correct for accesskit, however. I'd have to dig into it.

Reply:
The AccessKit adapter definitely doesn't own the window. The platform backends are supposed to deal gracefully with the adapter outliving the window, but still, the adapter _should_ be destroyed first. I'm not so sure about the surface. I'd think that if the surface owned an `Arc<Window>`, we'd see `Arc<Window>` as a parameter on `RenderSurface`, as I did with softbuffer's `Surface` when experimenting with that yesterday.

Reply:
Internally, wgpu stores the raw window in a boxed form. But yeah, this specific comment doesn't matter too much.